### PR TITLE
[Hotfix][Connector-V2-e2e][Spark] Corrected spark version

### PR DESCRIPTION
**Pull request by TyrantLucifer**
## Purpose of this pull request

Corrected spark version

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/TestContainerId.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
 @Getter
 public enum TestContainerId {
     FLINK_1_13(FLINK, "1.13.6"),
-    SPARK_2_4(SPARK, "2.4.3"),
+    SPARK_2_4(SPARK, "2.4.6"),
```
Comment (EricJoy2048) [inline seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/TestContainerId.java:30]: What's wrong with `2.4.3`?

**Review by hailin0 — APPROVED** [review]

**Review by ashulin — APPROVED** [review]
LGTM

File: seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/TestContainerId.java (hunk 1 above)
Comment (TyrantLucifer) [reply seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/TestContainerId.java:30]: Because image is changed.

Comment (ashulin) [reply seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/TestContainerId.java:30]: the spark docker's image version number is already 2.4.6

**Review by davidzollo — APPROVED** [review]
LGTM

**Pull request merged**